modules/orbidderBidAdapter.js

```diff
@@ -93,6 +93,9 @@ export const spec = {
       if (bidderRequest && bidderRequest.refererInfo) {
         referer = bidderRequest.refererInfo.page || '';
       }
+      if (bidRequest?.mediaTypes?.video) {
```

Comment:
I'm confused, on line 4 and 49 this bidder already makes it clear they do not support video

@dgirardi this pr seems to imply a feature request (bug?) in the ortb2 convertor, in which a bidder receives unsupported media types

Reply:
this is not using ortb converter. maybe it makes sense for core to remove unsupported media types, but on the other hand, adapters do not typically just dump the entirety of their input like this. Any modification of any kind to bid requests might break it for all we know.